FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java

```diff
@@ -0,0 +1,149 @@
+    @Override
+    public int getItemCount() {
+        return users.size();
+    }
+
+
+    // stores and recycles views as they are scrolled off screen
+    public class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
+        private ImageView ivPfp;
+        private ImageView ivAddFriend;
+        private TextView tvUsername;
+        private TextView tvLocation;
+        private boolean follows = false;
+
+        ViewHolder(View itemView) {
+            super(itemView);
+            ivPfp = itemView.findViewById(R.id.ivSearchPfp);
+            ivAddFriend = itemView.findViewById(R.id.ivAddFriend);
+            tvUsername = itemView.findViewById(R.id.tvSearchUsername);
+            tvLocation = itemView.findViewById(R.id.tvSearchLocation);
+            itemView.setOnClickListener(this);
+        }
+
+        @Override
+        public void onClick(View v) {
+            /*
+
+            // gets item position
+            int position = getAdapterPosition();
+            // make sure the position is valid, i.e. actually exists in the view
+            if (position != RecyclerView.NO_POSITION) {
+                // get the movie at the position, this won't work if the class is static
+                Post post = allPosts.get(position);
+                // create intent for the new activity
+                Intent intent = new Intent(context, PostDetailActivity.class);
+                intent.putExtra(Post.class.getSimpleName(), post);
+                context.startActivity(intent);
+            }*/
+
+        }
+
+        public void bind(ParseUser user) {
+            tvUsername.setText(user.getUsername());
+            tvLocation.setText(user.getString("city") + ", " + user.getString("state"));
+            RequestOptions requestOptions = new RequestOptions();
+            requestOptions = requestOptions.transforms(new CenterCrop(), new RoundedCorners(90));
+            ParseFile pfp = user.getParseFile("profilePhoto");
+            if (pfp != null){
+                Glide.with(context).applyDefaultRequestOptions(requestOptions).load(pfp.getUrl()).into(ivPfp);
+            }
+            else{
+                Glide.with(context).applyDefaultRequestOptions(requestOptions).load(context.getResources().getIdentifier("ic_baseline_face_24", "drawable", context.getPackageName())).into(ivPfp);
+            }
+
+            if(follows){
+                Glide.with(context)
+                        .load(R.drawable.ic_baseline_person_remove_24)
+                        .into(ivAddFriend);
+            }
+            else{
+                Glide.with(context)
+                        .load(R.drawable.ic_baseline_person_add_24)
+                        .into(ivAddFriend);
+            }
+
+            ivAddFriend.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    if(follows){
+                        //do something to database
```

Comment:
can you be more specific in terms of what you want to do the database? Like add/remove something

Reply:
removed  this comment!